2021q2/iwlwifi.adoc

```diff
@@ -0,0 +1,33 @@
+== Intel Wireless driver support
+
+Links: +
+link:https://wiki.freebsd.org/WiFi/Iwlwifi[iwlwifi status FreeBSD wiki page]  
+
+==== Newer Intel Wireless device support
+
+The Intel Wireless driver update project aims to bring support for
+newer chipsets along with mac80211 LinuxKPI compat code.
+The dual-licensed Intel driver code was ported in the past for the iwm(4)
+native driver and using the LinuxKPI compat framework allows us to use
+the driver directly with only very minor modifications.
+Multiple updates during development in the last year have shown that
+pulling in newer versions can be done in under 1-2 hours.
+
+After a break, part-time work resumed and the last LinuxKPI conflicts
```

Comment:
I'd make the first "and" in that sentence a comma instead, and I'd add a serial comma before the second "and" (next line).